src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -46,13 +44,9 @@ protected function doClone($var)
         $stub = null;                   // Stub capturing the main properties of an original item value
                                         // or null if the original value is used directly
 
-        if (!self::$hashMask) {
-            self::$gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
-            self::initHashMask();
+        if (!$gid = self::$gid) {
+            $gid = self::$gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
```

Comment:
Shouldn't `random_bytes` be better, especially as you are targetting master branch, which would be 4.0 (and thus need php >= 7.1) ?

Reply:
wouldn't provide anything, so better not change what's already fine